### DISCUSSION TITLE
无法和MCL通信

### DISCUSSION THREAD

**Discussion by ChocolateJin** [nonebot/discussions#34] [问答] [answered]
基于anaconda的prompt，MCL成功登录QQ并且能够从QQ获得信息，但nonebot无法和MCL通信
以下是我setting.yml、bot.py和.env.dev 的设置和运行情况
![image](https://user-images.githubusercontent.com/56467837/132680769-560ba662-dabc-4c58-909d-fd34d84fefb9.png)
![image](https://user-images.githubusercontent.com/56467837/132680768-7e74792e-4aa0-4536-9ca2-6a1e5abb641d.png)
![image](https://user-images.githubusercontent.com/56467837/132680820-d11c3520-f086-4bf3-a154-c392665bc686.png)
希望能够得到帮助！谢谢！

**Comment by mnixry** [comment, marked as answer]
~~注意打个码, 你QQ密码露出来了~~
目前NoneBot2暂时不支持MAH 2.0 (https://github.com/nonebot/nonebot2/pull/518) , 请暂时将MAH版本设定为 1.10.0, 然后[按照文档中的配置](https://v2.nonebot.dev/guide/mirai-guide.html)对MAH进行配置
不过, 我记得按照文档提供的指令, MCL应该是装不上MAH 2.x的啊?

**Reply by ChocolateJin** [reply]
谢谢，MAH和MCL是我的基友打包发给我的，本来是想给我省点事没想到反而出问题了，感谢您的解答，我现在就去试一下！
